pkg/bundle/internal/target/target.go

```diff
@@ -151,7 +149,7 @@ func (r *Reconciler) syncConfigMap(
 		return false, fmt.Errorf("failed to patch %s %s: %w", target.Kind, target.NamespacedName, err)
 	}
 
-	log.V(2).Info(fmt.Sprintf("synced bundle to namespace for target %s", target.Kind))
+	logf.FromContext(ctx).V(2).Info("synced bundle to namespace")
```

Comment:
note: We're not setting the target type in `pkg/bundle/bundle.go`

suggestion: is it worth adding the namespace name to this log line (and to  the equivalent for `syncSecret`)? Otherwise this might be quite a spammy log line with little context.

Reply:
The log-context should be covered by:

https://github.com/erikgb/trust-manager/blob/8779ee435fe0303e265abadb9417ed520ee8cd72/pkg/bundle/bundle.go#L282-L283